.github/workflows/haskell.yml

```diff
@@ -51,6 +51,7 @@ jobs:
           - 8.10.7+exe
           - 9.0.1+exe
           - 9.2.1
+          - 9.4.1
```

Comment:
We need add a corresponding entry in the `include` section below. CIs are not getting started because of this issue.

Also we can bump the minor version to 9.4.2 .

Reply:
Ah, I missed that this is only the `name`. Should be fixed now.